### ISSUE TITLE
How to install the google-translate-flag-dropdown on my website.

### ISSUE THREAD

**Issue by tradegong** [hc0503/google-translate-flag-dropdown#9]
Good day all! This plugin (google-translate-flag-dropdown) looks great. However, I am not quite a profi in this. Can someone here be so nice as to put me through regarding how ti install and make it work on my site?
How do I set it up?
My understanding is that I will copy all the codes together one after the other and place at the header section of my site, then create the flag images and place them in any folder anywhere on the site, while linking to the flag images. Am I by any way correct?
I would really appreciate some help with a step-by-step guide please. Tnx in advance.

**Comment by hc0503** [comment]
@tradegong , You're right, follow the steps of Readme, but remember this is old example with old versions.

**Comment by tradegong** [comment]
Okay.  Tnx very much for your prompt reply.  Nevertheless, where can I download the  current version with current instructions/ readme?
Does it work with every type of website or only on specific types of websites?
I look forward to your response. 
Tnx

**Comment by tradegong** [comment]
Good day sir!

<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="flag-icon flag-icon-af"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

Please, if I may ask, where in the above example for "Afrikaans" is the link for my flag file or flag file path? I am almost asuming that i have to take out the ""flag-icon flag-icon-af"" above and interchange it with the path to my file as as shown in the modidcation below, asuming that my filepath is "tradegong.fr/google-translate-flag-dropdown-main/flags/af.png":

<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="tradegong.fr/google-translate-flag-dropdown-main/flags/af.png"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

I believe the above configuration of the flag icon path is correct? If not, could you be so kind to point me in the direction by indicating where the flag file path should be inserted? 
I look forward to your kind assistance.
Regards,
Marsh

**Comment by tradegong** [comment]
Good day sir!

<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="flag-icon flag-icon-af"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

Please, if I may ask, where in the above example for "Afrikaans" is the link for my flag file or flag file path? I am almost asuming that i have to take out the ""flag-icon flag-icon-af"" above and interchange it with the path to my file as as shown in the modidcation below, asuming that my filepath is "tradegong.fr/google-translate-flag-dropdown-main/flags/af.png":
My configuration below in next comment:

**Comment by tradegong** [comment]
<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="tradegong.fr/google-translate-flag-dropdown-main/flags/af.png"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

I believe the above configuration of the flag icon path is correct? If not, could you be so kind to point me in the direction by indicating where the flag file path should be inserted? 
I look forward to your kind assistance.
Regards,
Marsh

**Comment by tradegong** [comment]
<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="flag-icon flag-icon-af"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

to:

<select class="selectpicker" data-width="fit" onchange="translateLanguage(this.value);">
  <option data-content='<span class="tradegong.fr/google-translate-flag-dropdown-main/flags/af.png"></span> Afrikaans' value="Afrikaans">Afrikaans</option>
  ...
</select>

**Comment by tradegong** [comment]
For whatever reason, my complete message is mutilated. That is why I have written multiple messages..
Where should my file path come in please?

**Comment by tradegong** [comment]
Should I exchange this line:

""flag-icon flag-icon-af"" 

with this line (file-path:

"tradegong.fr/google-translate-flag-dropdown-main/flags/af.png"  ?? Would that be correct??

**Comment by tradegong** [comment]
Tnx
